scripts/tvm_cli/build.sh

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env bash
+# Copyright 2020 Autoware Foundation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+IMAGE_NAME="autoware/model-zoo-tvm-cli"
+TAG_NAME="local"
+
+function usage() {
+    echo "Usage: $0 [OPTIONS]"
+    echo "    -h,--help              Display the usage and exit."
+    echo "    -i,--image-name <name> Set docker images name."
+    echo "                           Default: $IMAGE_NAME"
+    echo "    -t,--tag <tag>         Tag use for the docker images."
+    echo "                           Default: $TAG_NAME"
+    echo ""
+}
+
+OPTS=$(getopt --options hi:t: \
+         --long help,image-name:,tag: \
+         --name "$0" -- "$@")
+eval set -- "$OPTS"
+
+while true; do
+  case $1 in
+    -h|--help)
+      usage
+      exit 0
+      ;;
+    -i|--image-name)
+      IMAGE_NAME="$2"
+      shift 2
+      ;;
+    -t|--tag)
+      TAG_NAME="$2"
+      shift 2
+      ;;
+    --)
+      if [ -n "$2" ];
+      then
+        echo "Invalid parameter: $2"
+        exit 1
+      fi
+      break
+      ;;
+    *)
+      echo "Invalid option"
+      exit 1
+      ;;
+  esac
+done
+
+SCRIPT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+FROM_ARG="ubuntu:18.04"
+if [[ -d "/proc/driver/nvidia" ]]; then
```

Comment:
I would recommend making this an option rather than checking the current user's system. This container can be built without the person building the image having an NVIDIA graphics card, yeah?

Reply:
Thanks!